ethereumj-core/src/main/java/org/ethereum/datasource/DbSettings.java

```diff
@@ -0,0 +1,33 @@
+package org.ethereum.datasource;
+
+/**
+ * Defines configurable database settings
+ *
+ * @author Mikhail Kalinin
+ * @since 26.04.2018
+ */
+public class DbSettings {
+
+    public static final DbSettings DEFAULT = new DbSettings();
+
+    int maxOpenFiles = 32;
```

Comment:
Maybe add 2 more vars as statics above and assign here from them?
Don't like a way magic constants are passed.

Reply:
Agree. Don't like them either :)